numerals.js

```diff
@@ -0,0 +1,57 @@
+function letters(number, symbol){
+  rom_str = ''
+  for(var i = 0; i < number; i ++){
+    rom_str += symbol
+  }
+  return rom_str
+}
+
+function digit(input, pow_10, symbols){
```

Comment:
A bit difficult to understand what is happening in this function. Maybe better function naming and separating out some functionality into smaller methods would make it more clear. Try to keep functions smaller than ~8 lines